pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
Would I, as a user, want multiple groups?  Technically the only reason we have uid is as a fs uid, but we don't call it that.  The allocator should try to be generic - the strategy is specific.  So the strategy is saying "I want a unique group AND user AND selinux label for every project SO THAT filesystems don't overlap".  If the manifestation of that is a uid range and selinux labels and now a group.  I think this annotation should probably be supplemental groups and we should use the first one as the FS group.  But not a range, because we want to allow admins to stitch together groups if necessary.